extensions/amp-ad/0.1/concurrent-load.js

```diff
@@ -49,37 +45,18 @@ export function allowRenderOutsideViewport(element, win) {
 }
 
 /**
- * Decrements loading ads count used for throttling.
- * @param {number|string} timerId of timer returned from incrementLoadingAds
- * @param {!Window} win
- */
-export function decrementLoadingAds(timerId, win) {
-  timerFor(win).cancel(timerId);
```

Comment:
Yes, previously we incremented at start of layoutCallback and decrement once we know if creative is AMP.  Now, we only increment after knowing if that creative is not AMP.